hack/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Do we need this because you are using old version golagn?

Reply:
those changes are not expected. I will revert them back